runtime/io.c

```diff
@@ -519,10 +519,11 @@ CAMLprim value caml_ml_open_descriptor_in(value fd)
   return caml_alloc_channel(chan);
 }
 
-CAMLprim value caml_ml_open_descriptor_out(value fd)
+CAMLprim value caml_ml_open_descriptor_out(value fd, value buffered)
```

Comment:
There's a lot of code out there which expects `caml_ml_open_descriptor_out` to be an arity-1 primitive

Reply:
I don't think the C code needs changing.  Just amend stdlib/stdlib.ml:
```
let stderr = open_descriptor_out 2
external set_buffered : t -> bool -> unit = "caml_ml_set_buffered"
let _ = set_buffered stderr false
```